iothub/device/src/InternalClient.cs

```diff
@@ -600,12 +605,7 @@ public Task RejectAsync(string lockToken, CancellationToken cancellationToken)
         /// <returns>The lock identifier for the previously received message</returns>
         public Task RejectAsync(Message message)
         {
-            if (message == null)
-            {
-                throw new ArgumentNullException(nameof(message));
-            }
-
-            return RejectAsync(message.LockToken);
+            return message == null ? throw new ArgumentNullException(nameof(message)) : RejectAsync(message.LockToken);
```

Comment:
nit: multiline here?

Reply:
I will make a note and update it in another PR.